tests/test_db_function.py

```diff
@@ -53,3 +58,49 @@ def test_non_blob_columns_mutable(db_session):
         db_session.commit()
     except sqlalchemy.exc.InternalError:
         raise AssertionError
+
+
+def test_create_scores_for_application_sub_crit():
+    """test_create_scores_for_application_sub_crit Tests we can create
+    score records in the scores table in the appropriate format."""
+
+    picked_row = get_random_row(AssessmentRecord)
+    application_id = picked_row.application_id
+    sub_criteria_id = "app-info"
+    
+    assessment_payload = {
+        "application_id": application_id,
+        "sub_criteria_id": sub_criteria_id,
+        "score": 3,
+        "justification": "bang average",
+        "user_id": "test"
+    }
+    score_metadata = create_score_for_application_sub_crit(**assessment_payload)
+
+    assert len(score_metadata) == 7
+    assert score_metadata["date_created"]
```

Comment:
Might be better to use ` "data_created" in score_metdata ` so its more clear what the check is.